src/mercedestrenz/data.py

```diff
@@ -22,21 +23,21 @@ def load_sample_mercedes_listings()-> pd.DataFrame():
 
 # Author: Kelly Wu
 # Date: 2023-01-12
-def listing_search(max_price=30000, region = "any", model = "any", sort_feature = "odometer", ascending = True)-> pd.DataFrame():
+def listing_search(data, budget=[0, np.Inf], model = "any", sort_feature = "odometer", ascending = True)-> pd.DataFrame():
```

Comment:
nitpick:  Since we import Numpy just for it's infinity value, maybe try to use base python which after some googling I think `float("inf")` gives the same behaviour as `np.Inf` without needing Numpy as dependency!